cmd/all-in-one/Dockerfile

```diff
@@ -21,6 +21,7 @@ EXPOSE 14250
 # Web HTTP
 EXPOSE 16686
 
+COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
```

Comment:
This is not everything, we need also these lines 

https://github.com/jaegertracing/jaeger/blob/master/cmd/collector/Dockerfile#L1-L2

Reply:
aha, my bad. raised the PR in a hurry. Lesson learned. 
thanks for the comment.

Reply:
Also please move this `COPY` statement right after the `from scratch`, then it's ready to be merged.